src/common/constants.lua

```diff
@@ -3,7 +3,8 @@ local constants = {}
 constants.MAX_UNDERNAME_LENGTH = 61
 constants.UNDERNAME_DOES_NOT_EXIST_MESSAGE = "Name does not exist in the ANT!"
 constants.INVALID_ARWEAVE_ID_MESSAGE = "Invalid Arweave ID"
-constants.MIN_TTL_SECONDS = 900
+constants.MIN_TTL_SECONDS = 60
+constants.DEFAULT_TTL_SECONDS = 900
 constants.MAX_TTL_SECONDS = 3600
```

Comment:
i think this should be 24 hours

Reply:
https://github.com/ar-io/ar-io-ant-process/pull/66/commits/db517ee0753adf8517f7ac3fa617ecb7376368a0